charts/cluster-api-cluster-openstack/README.md

```diff
@@ -49,28 +50,25 @@ spec:
           region: en-west-1
           failureDomain: eu-west-1a
           externalNetworkID: dadfef54-d1c5-447a-8933-f515eeadd822
-        cluster:
-          taints:
-          - key: node.cilium.io/agent-not-ready
-            effect: NoSchedule
-            value: 'true'
         api:
           allowList:
           - 123.45.67.89
           certificateSANs:
           - kubernetes.my-domain.com
         controlPlane:
+          version:  v1.30.2
           replicas: 3
+          registerNodes: true
```

Comment:
Note to self: when the next PR goes in, you'll need to cross reference this change with the actual change.

Reply:
oooh wait,m that needs removing. I'm not adding register nodes anymore as it's not achieving what I wanted. Let me remove that

Reply:
Removed it.